saleor/payment/gateways/authorize_net/__init__.py

```diff
@@ -20,6 +16,16 @@
     PaymentMethodInfo,
 )
 
+# vendored dependencies
+sys.path.append(os.path.join(os.path.dirname(__file__), "vendored"))
```

Comment:
The [`setup.py`](https://github.com/AuthorizeNet/sdk-python/blob/11f5f69dc965c72c0e96d4bc6a4f98ca84727fa3/setup.py#L1) from AuthorizeNet could be used instead, Python will take care of the rest

Docs:
https://python-poetry.org/docs/dependency-specification/#path-dependencies

Reply:
We don't rely on poetry for running Saleor and we don't require poetry for installation.